my-project/src/Controller/NavController.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
```

Comment:
add header to all new custom files, exaple below:
/**
 * What the file does
 *
 * @category   Bootstrap
 * @package    Virtua_Internship
 * @copyright  Copyright (c) 2018 Virtua (http://www.wearevirtua.com)
 * @author     Name Lastname <contact@wearevirtua.com>
 */